docs/sources/config/default.cfg

```diff
@@ -116,9 +116,18 @@ flip = False
 # Rotation of the camera: 0, 90, 180 or 270, can be different for preview and capture (list of integers accepted)
 rotation = 0
 
-# Resolution for camera captures (preview will have same aspect ratio)
+# Resolution for camera captures
 resolution = (1934, 2464)
 
+# Preview Resolution for camera captures (Width, Height) or None to get the resolutions from Camera capabilities
+preview_resolution = None
+
+# Use V4L2 instead of GSTREAMER
+use_v4l2 = False
+
+# Custom FourCC (can be YUY2 or MJPG or any other fourcc or None to use the default)
+custom_fourcc = None
+
```

Comment:
I'm not very fan to add opencv only settings in the config (I don't think we have this right now)

Reply:
what do you suggest ? Perhaps create a specific bloc in configuration for opencv [OPENCV] in the ini file ?

Reply:
I've already finished a refactor with a dedicated [OPENCV] configuration section. ^^